package.json

```diff
@@ -76,6 +76,7 @@
     "pivottable": "^2.15.0",
     "plotly.js": "1.41.3",
     "prop-types": "^15.6.1",
+    "qs": "^6.7.0",
```

Comment:
You had to commit an updated `package-lock.json` file as well.

Reply:
Hmm... I wonder how that slipped. Should I do this in a new PR?

Reply:
Yes.